src/qml/qgismobileapp.qml

```diff
@@ -3094,6 +3094,108 @@ ApplicationWindow {
     visible: false
   }
 
+
+  Image {
```

Comment:
Wondering whether it makes sense to move the whole thing in a separate component file?

Reply:
I'm worried having it in a separate component file will make it too easy to find out :)

Reply:
If someone looks at the source I think he deserves to know